code/modules/mob/living/simple_animal/eldritch_demons.dm

```
@@ -0,0 +1,272 @@
+	del_on_death = TRUE
+	deathmessage = "implodes into itself"
+	///Innate spells that are supposed to be added when a beast is created
+	var/list/spells_to_add
+
+/mob/living/simple_animal/hostile/eldritch/Initialize()
+	. = ..()
+	add_spells()
+
+/**
+  * Add_spells
+  *
+  * Goes through spells_to_add and adds each spell to the mind.
+  */
+/mob/living/simple_animal/hostile/eldritch/proc/add_spells()
+	for(var/spell in spells_to_add)
+		AddSpell(new spell())
+
+/mob/living/simple_animal/hostile/eldritch/raw_prophet
+	name = "Raw Prophet"
+	real_name = "Raw Prophet"
+	desc = "Abomination made from severed limbs."
+	icon_state = "raw_prophet"
+	status_flags = CANPUSH
+	icon_living = "raw_prophet"
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	maxHealth = 50
+	health = 50
+	sight = SEE_MOBS|SEE_OBJS|SEE_TURFS
+	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/targeted/telepathy/eldritch)
+
+/mob/living/simple_animal/hostile/eldritch/raw_prophet/Login()
+	. = ..()
+	client?.view_size.setTo(11)
+
+/mob/living/simple_animal/hostile/eldritch/armsy
+	name = "Terror of the night"
+	real_name = "Armsy"
+	desc = "Abomination made from severed limbs."
+	icon_state = "armsy_start"
+	icon_living = "armsy_start"
+	maxHealth = 200
+	health = 200
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	move_resist = MOVE_FORCE_OVERPOWERING+1
+	///Previous segment in the chain
+	var/mob/living/simple_animal/hostile/eldritch/armsy/back
+	///Next segment in the chain
+	var/mob/living/simple_animal/hostile/eldritch/armsy/front
+	///Your old location
+	var/oldloc
+	///Allow / disallow pulling
+	var/allow_pulling = FALSE
+	///How many arms do we have to eat to expand?
+	var/stacks_to_grow = 5
+	///Currently eaten arms
+	var/current_stacks = 0
+
+/mob/living/simple_animal/hostile/eldritch/armsy/New(spawn_more = TRUE,len = 6)
+	. = ..()
+	len = max(3,len) //code breaks below 3, let's just not allow it.
```

```suggestion
	if(len < 3)
		stack_trace("Eldritch Armsy created with invalid len ([len]). Reverting to 3.")
		len = 3 //code breaks below 3, let's just not allow it.
```
We want to avoid silent errors.